saleor/settings.py

```diff
@@ -541,3 +548,14 @@ def get_host():
 JWT_TOKEN_SECRET = os.environ.get("JWT_TOKEN_SECRET", "saleor")
 if not DEBUG:
     JWT_VERIFY_EXPIRATION = True
+
+
+if (
+    not DEBUG
```

Comment:
Not if this is needed, this warning might be useful with `DEBUG=True` as well.

Reply:
Maybe we shouldn't raise but only warn then?

Reply:
Feature works out of the box for development since `localhost` and `127.0.0.1` are default for `ALLOWED_CLIENT_HOSTS` (but removing debug line would raise the exception for them). Assuming no production environment is running with DEBUG on, I think it's preferred to leave it as is, it's better to have an exception about this rather than warning in production env (and if we remove the debug it would have to be changed to warning).